workflow-templates/test-package.yaml

```diff
@@ -28,6 +30,7 @@ jobs:
         run: dart pub get
       - name: Check formatting
         run: dart format --output=none --set-exit-if-changed .
+        if: always() && steps.install.outcome == 'success'
```

Comment:
That doesn't actually change anything, if any of the steps preceding this one fail, it makes no sense to run this either. But Ok for me if this is just for symmetry.